silk-react-components/src/HierarchicalMapping/Components/MappingRule/ErrorView.jsx

```diff
@@ -22,58 +24,57 @@ const ErrorCause = ({errorCause}) => (
     </ul>
 );
 
-const ErrorIssue = ({errorCause}) => (
+export const ErrorIssue = ({ errorCause }) => (
     <ul className="ecc-hierarchical-mapping-error-list">
-        {_.map(errorCause, ({message}) => (
+        {_.map(errorCause, ({ message }) => (
             <li>
                 <p>{message}</p>
             </li>
         ))}
     </ul>
 );
 
-const ErrorView = React.createClass({
-    propTypes: {
-        title: React.PropTypes.string,
-        detail: React.PropTypes.string,
-        cause: React.PropTypes.object, // it may contain a list for errors with title and detail itself
-        issues: React.PropTypes.object, // it may contain a list for errors with title and detail itself, too
-    },
-    componentDidMount() {},
-    getInitialState() {
-        return {
-            errorExpanded: false,
-        };
-    },
+export class ErrorView extends React.Component {
+    static propTypes = {
+        title: PropTypes.string,
+        detail: PropTypes.string,
+        cause: PropTypes.array, // it may contain a list for errors with title and detail itself
+        issues: PropTypes.array, // it may contain a list for errors with title and detail itself, too
+    };
+    
+    state = {
+        errorExpanded: false
```

Comment:
There are a couple of lint errors like `comma-dangle` and `no-traling-spaces`. Should be reviewed on all files.

Reply:
I fixed some kind of issues by running `yarn lint --fix`, but the eslint warnings and errors are too much for fixing that and it's not critical, I think ?)